Jenkinsfile_CNP

```diff
@@ -0,0 +1,24 @@
+#!groovy
+@Library("Infrastructure")
+
+import uk.gov.hmcts.contino.GradleBuilder
+import uk.gov.hmcts.contino.Kubectl
+import uk.gov.hmcts.contino.DockerImage
+import uk.gov.hmcts.pipeline.TeamConfig
+
+def type = "java"
+def product = "bulk-scan"
+def component = "pay-processor"
```

Comment:
because `payment-processor` is too long i presume, right?

Reply:
yes that's correct